main.go

```diff
@@ -1,181 +1,135 @@
 				},
 				cli.Command{
 					Name:        "individual",
-					Action:      actions.GenerateIndividualChangelog(ghClient, os.Stdout),
+					Action:      actions.GenerateIndividualChangelog(ghclient, os.Stdout),
 					Usage:       "deisrel changelog individual <repo-name> <old-release> <sha> <new-release>",
 					Description: "Generate a changelog entry for an changes on an individual repository, from a specified old release through a specified git SHA. The release will be called the specified new release in the changelog's title",
 				},
 			},
 		},
-		cli.Command{
-			Name: "helm-stage",
-			Flags: []cli.Flag{
-				cli.StringFlag{
-					Name:  actions.TagFlag,
-					Value: "",
-					Usage: "The Docker tag to apply for all images. If empty, defaults to 'git-$SHORT_SHA' for each respective component",
-				},
-				cli.StringFlag{
-					Name:  actions.PullPolicyFlag,
-					Value: "IfNotPresent",
-					Usage: "The 'imagePullPolicy' value to set on each image",
-				},
-				cli.StringFlag{
-					Name:  actions.OrgFlag,
-					Value: "deisci",
-					Usage: "The docker repository organization to set on each image",
-				},
-				cli.StringFlag{
-					Name:  actions.RepoFlag,
-					Value: "charts",
-					Usage: "The GitHub repo name to grab chart from",
-				},
-				cli.StringFlag{
-					Name:  actions.RefFlag,
-					Value: "master",
-					Usage: "Optional ref to add to GitHub repo request (can be SHA, branch or tag)",
-				},
-				cli.StringFlag{
-					Name:  actions.GHOrgFlag,
-					Value: "deis",
-					Usage: "The GitHub org to find repo in",
-				},
-				cli.StringFlag{
-					Name:  actions.StagingDirFlag,
-					Value: "",
-					Usage: "The location to stage chart files",
-				},
-			},
-			Subcommands: []cli.Command{
-				cli.Command{
-					Name:        "e2e",
-					Action:      actions.HelmStageE2E(ghClient),
-					Description: "Stages workflow-dev-e2e, amending with $WORKFLOW_RELEASE_SHORT if defined",
-				},
-				cli.Command{
-					Name:        "router",
-					Action:      actions.HelmStageRouter(ghClient),
-					Description: "Stages router-dev, amending with $WORKFLOW_RELEASE_SHORT if defined",
-				},
-				cli.Command{
-					Name:        "workflow",
-					Action:      actions.HelmStageWorkflow(ghClient),
-					Description: "Stages workflow-dev, amending with $WORKFLOW_RELEASE_SHORT if defined",
-				},
-			},
-			Description: `Stages chart files into staging.
-			To amend files with values pertinent to a release, user must export the following env variables:
-			$WORKFLOW_RELEASE=<full_semver_release_string>, i.e. 'v1.0.0-alpha1'
-			$WORKFLOW_RELEASE_SHORT=<short_form_release_string>, i.e. 'alpha1'`,
-		},
 	}
 
-	if err := app.Run(os.Args); err != nil {
-		os.Exit(1)
+	app.Action = func(c *cli.Context) error {
+		res := make(map[string]interface{})
+
+		if c.NArg() < 2 {
+			return cli.NewExitError("A params and a versions file is required", 1)
```

Comment:
should it be called `versions file` or something like `repo-to-component map file`?